orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -260,13 +305,14 @@ at::Tensor view(const at::Tensor& self, at::IntArrayRef size) {
   ORT_LOG_FN(self, size);
 
   auto& invoker = GetORTInvoker(self.device());
+  auto shape = at::infer_size_dv(size, self.numel());
```

Comment:
why it can't be directly passed to reshape_copy?

Reply:
There is a type mismatch, the return type of infer_size_dv is at:DimVector and the return type of infer_size was std::vector. If I just pass the shape it says cannot convert argument 3 from 'at::DimVector' to 'std::vector<int64_t,std::allocator<int64_t>>'